### ISSUE TITLE
Should we move the atlas in /home/perinedo/Projects/Atlases/M-CRIB/M-CRIB_for_MRtrix_5ttgen_neonatal to code directory?

### ISSUE THREAD

**Issue by Dlasic7** [HIIM-PERINEDO/zagreb_dhcp#41]
It would be good to have the atlas in code directory because else we have to hardcode the path of the atlas (/home/perinedo/Projects/Atlases/M-CRIB/M-CRIB_for_MRtrix_5ttgen_neonatal). Maybe put it into the top directory of our repo under directory "atlases"?

**Comment by finnlennartsson** [comment]
No, I think we will have to keep it outside of our github as there can be copyright issues with this. 
An alternative would be to put it $studydir/atlases. Then we can use relative paths in relation to $studydir.

**Comment by finnlennartsson** [comment]
Or we can put it into `/code` but not into `/code/zagreb_dhcp`
